authority/ssh.go

```diff
@@ -258,7 +258,7 @@ func (a *Authority) SignSSH(ctx context.Context, key ssh.PublicKey, opts provisi
 // RenewSSH creates a signed SSH certificate using the old SSH certificate as a template.
 func (a *Authority) RenewSSH(ctx context.Context, oldCert *ssh.Certificate) (*ssh.Certificate, error) {
 	if oldCert.ValidAfter == 0 || oldCert.ValidBefore == 0 {
-		return nil, errs.BadRequest("renewSSH: cannot renew certificate without validity period")
+		return nil, errs.BadRequest("cannot renew a certificate without validity period")
```

Comment:
it's not really necessary to add these here. I don't think error messages are expected to be grammatically correct. They're just meant to be informative. The extra `a` doesn't convey any info that isn't already there.

Reply:
Let's keep it as it is, I think it looks more natural and this will be displayed to the user.